libraries/chain/hardfork.d/999.hf

```diff
@@ -0,0 +1,4 @@
+// Placeholder HF for affiliate reward system
+#ifndef HARDFORK_999_TIME
+#define HARDFORK_999_TIME (fc::time_point_sec( 2000000000 ))
```

Comment:
Shouldn't this PR contain the real HF time?

Reply:
The real HF date is still pending finalization ..